activities/Chess.activity/js/chessGame.js

```diff
@@ -458,18 +468,21 @@ var ChessGame = {
 
         var move = this.state.move(source, target);
 
+        // console.log(`Move: ${source}-${target}, Flags: ${move.flags}`);
         // illegal move
         if (move.flags === 0) return 'snapback';
 
-        if (move.flags === 7) {
+        if (move.flags === 7 || move.flags === 15 || move.flags === 23 || move.flags === 39) {
+          console.log("Game won by player");
           this.game_won = true;
-        } else if ((!(move.flags & (1 << 1)) && move.flags & (1 << 2)) || move.flags & (1 << 6)) {
+        } else if ( (!(move.flags & 2 )) && ((move.flags & 4 ) || move.flags & (64))) {
```

Comment:
This line do the same test than before

Reply:
it's not supposed to be condition 1 not true AND condition two OR condition three true. It's supposed to be Condition one not true AND ( condition two or condition 3 is true).

Reply:
Ok got it.